citi-onboarding/iJunior-onepage
# Feature/frontend-development

## What I did

- Created the front-end development section
- Inserted cards in components
   - responsives cards
- Inserted development folder in views

| Mockup | Real |
|--------|------|
| <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/123169441-b17a0180-d44f-11eb-850a-e94a6dfadb4d.png"> | <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/123170247-ac698200-d450-11eb-98a4-f01cf70258ca.png"> |

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/frontend-development`
   1. `git pull origin feature/frontend-development`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `client/src/views and client/src/components`

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 2h13min

**Resolves:** # Frontend of the "Development" section
